core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -532,93 +535,94 @@ func (o *evmTxStore) InsertTx(etx *Tx) error {
 ) RETURNING *`
 	var dbTx DbEthTx
 	dbTx.FromTx(etx)
-	err := o.q.GetNamed(insertEthTxSQL, &dbTx, &dbTx)
+
+	query, args, _ := o.q.BindNamed(insertEthTxSQL, &dbTx)
```

Comment:
Unchecked error. More down below. Is the linter flagging these? It should be.

Reply:
Linter is not catching this. Added error handling.